### ISSUE TITLE
Not able to send confirmation email after cloning

### ISSUE THREAD

**Issue by pranavm2015** [BuildmLearn/University-Campus-Portal-UCP#3]

**Comment by mukeshkharita** [comment]
Can someone provide a description of this issue?

**Comment by pranav15197** [comment]
Have u added a local settings file with the SMTP details?

**Comment by mukeshkharita** [comment]
@pranav15197 Yes I have added all the SMTP details on settings_local.py file.
So How can I send confirmation email after cloning

**Comment by pranav15197** [comment]
Could you elaborate a bit more about the issue that you are facing

- Do you see any error raised in the terminal or in the browser itself?
- Do you get the email successfully sent, message but are still not able to receive any email?

**Comment by pranavm2015** [comment]
It gives the following error ( code - 403 )

CSRF verification failed. Request aborted.

**Comment by hao555sky** [comment]
Me too, I can not receive the confirmation email. It is no action for a long time when i click register button. Then I closed it.
